com.unity.render-pipelines.universal/Runtime/Documentation.cs

```diff
@@ -1,28 +1,28 @@
-using System;
 using System.Diagnostics;
 
-
 namespace UnityEngine.Rendering.Universal
 {
     [Conditional("UNITY_EDITOR")]
-    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Enum, AllowMultiple = false)]
-    internal class URPHelpURLAttribute : HelpURLAttribute
+    internal class URPHelpURLAttribute : CoreRPHelpURLAttribute
     {
-        //This must be used like
-        //[URPHelpURLAttribute("some-page")]
         public URPHelpURLAttribute(string pageName)
-            : base(Documentation.GetPageLink(pageName))
+            : base(pageName, Documentation.packageName)
         {
         }
     }
 
-
-    //Need to live in Runtime as Attribute of documentation is on Runtime classes \o/
-    class Documentation : DocumentationInfo
+    internal class Documentation : DocumentationInfo
     {
-        internal static string GetPageLink(string pageName)
-        {
-            return $"https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@latest/index.html?subfolder=/manual/{pageName}.html";
-        }
+        /// <summary>
+        /// The name of the package
+        /// </summary>
+        public const string packageName = "com.unity.render-pipelines.universal";
+
+        /// <summary>
+        /// Generates a help url for the given package and page name
+        /// </summary>
+        /// <param name="pageName">The page name</param>
+        /// <returns>The full url page</returns>
+        public static string GetPageLink(string pageName) => GetPageLink("com.unity.render-pipelines.universal", pageName);
```

Comment:
should it be GetPageLink(packageName, pageName) ?

Reply:
yeah, thanks! Last refactor that I didn't notice :)